tensorflow/docs_src/community/documentation.md

```diff
@@ -4,12 +4,24 @@ We welcome contributions to the Tensorflow documentation from the community.
 This document explains how you can contribute to that documentation. In
 particular, this document explains the following:
 
-- Where the documentation is located.
-- How to make conformant edits.
-- How to build and test your documentation changes before you submit them.
-
-You can view Tensorflow documentation on tensorflow.org, and you can view and
-edit the raw files on Github.
+* Where the documentation is located.
+* How to make conformant edits.
+* How to build and test your documentation changes before you submit them.
+
+You can view Tensorflow documentation on https://www.tensorflow.org, and you
+can view and edit the raw files on Github. We're publishing our docs on Github
+so everybody can contribute. Whatever gets checked in tensorflow/docs_src will
+be published soon after on https://www.tensorflow.org. 
+
+Republishing TensorFlow documentation in different forms is absolutely allowed,
+but we unlikely to accept anything other than the sources into our repository.
```

Comment:
It's vague here about "the sources".  Perhaps, "we **are** unlikely to accept anything but raw markdown source into docs_src."

Might add a paragraph break here, too.